sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -751,6 +751,25 @@ dynamic parameter for the buffer can be registered with all the nodes which
 use the buffer as a parameter. Then a single `dynamic_parameter::update()` call
 will maintain the graphs data dependencies.
 
+===== Whole Graph Update
+
+A graph in the executable state can have all of its nodes inputs & outputs updated
+using the `command_graph<graph_state::executable>::update()` method. This method takes
+a graph in the modifiable state and updates the executable graph to use the node inputs &
+outputs of the modifiable graph.
+
+The modifiable graph must have the same topology as the graph originally used to
+create the executable graphs, with the nodes targeting the same devices and
+added in the same order.
```

Comment:
> nodes targeting the same devices

This is probably a bit confusing given that multi-device graphs, AKA per node devices are still under discussion. Probably this should refer to the device matching the one used on graph construction.

Reply:
I updated the wording. Does it make more sense now?

Reply:
Yep looks good!